lib/ISAACConfig.cmake

```diff
@@ -219,7 +219,7 @@ FIND_PACKAGE_HANDLE_STANDARD_ARGS(ISAAC
                                         ISAAC_INCLUDE_DIRS
                                         JANSSON_LIBRARIES
                                         JANSSON_INCLUDE_DIRS
-                                        CMAKE_THREAD_LIBS_INIT
+                                        Threads_FOUND
```

Comment:
Could you make a separate PR for this one line change. We need to know if this is a bug and if we need to mark the change as bug to be able to write a changelog.